cfgov/v1/models/base.py

```diff
@@ -77,6 +81,21 @@ class CFGOVPage(Page):
             'Maximum size: 4096w x 4096h.'
         )
     )
+    schema_json = JSONField(
+        null=True,
+        blank=True,
+        verbose_name='Schema JSON',
+        help_text=mark_safe(
```

Comment:
Instead of adding this lengthy help text and having it get squished in the sidebar, you could use a MultiFieldPanel with a HelpPanel (if a bit too spaced out):

![image](https://user-images.githubusercontent.com/654645/79791820-eb1b1e00-831b-11ea-9783-7ae3429dade4.png)

Reply:
Eh, I don't mind calling less attention to the field. It should be avoided most of the time.